lab/experiments/ThemeTest.vue

```diff
@@ -310,6 +387,23 @@
 							@image-uploader:change="setImages"
 						/>
 					</div>
+					<m-divider />
+					<div :class="$s.TypographyPreview">
+						<m-text
+							v-for="variant in ['headline', 'title', 'paragraph', 'label']"
+							:key="variant"
+							:variant="variant"
+						>
+							{{ variant }}
+						</m-text>
+						<m-text
```

Comment:
Why have a size control for each type style? That's not a requirement we need to support

Reply:
Ah, I see now. I was getting this confused with some of the sliders above. I understand now, no concerns.